python/lsst/meas/base/plugins.py

```diff
@@ -68,6 +71,103 @@
                     TransformClass=bl.ApertureFluxTransform, executionOrder=BasePlugin.FLUX_ORDER)
 
 # --- Single-Frame Measurement Plugins ---
+class SingleFrameVarianceConfig(SingleFramePluginConfig):
+        scale = lsst.pex.config.Field(dtype=float, default=5.0, optional=True,
+                                      doc="scale factor to apply to shape for aperture")
+        mask = lsst.pex.config.ListField(doc="mask planes to ignore", dtype=str,
+                                         default=["DETECTED", "DETECTED_NEGATIVE"])
+
+@register("base_Variance")
+class SingleFrameVariancePlugin(SingleFramePlugin):
+    '''
+    Calculate the median variance within a Footprint scaled from the object shape so
+    the value is not terribly influenced by the object and instead represents the
+    variance in the background near the object.
+    '''
+
+    ConfigClass = SingleFrameVarianceConfig
+
+    @classmethod
+    def getExecutionOrder(cls):
+        return cls.FLUX_ORDER
+
+    def __init__(self, config, name, schema, metadata):
+        SingleFramePlugin.__init__(self, config, name, schema, metadata)
+        self.varValue = schema.addField(name + '_value', type="D", doc="variance at object position")
+        self.varFlag = schema.addField(name + '_flag', type="Flag", doc="Set to 1 for any fatal Failure")
+
+    def measure(self, measRecord, exposure):
+        # Promote the bounding box of the footprint to type D to ensure
+        # the ability to compare (they may be of mixed types I and D)
```

Comment:
This comment appears too early -- it refers to line 103, but is immediately followed by line 102 to which it is not relevant.

Also, I think it would be helpful to specify what you want to compare it with.